bentoml_cli/containerize.py

```diff
@@ -6,9 +6,12 @@
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
-    F = t.Callable[..., t.Any]
     from click import Group
 
+    from bentoml._internal.bento import BentoStore
+
+    F = t.Callable[..., t.Any]
```

Comment:
where is this used?

Reply:
it is not, but I will remove it in a separate PR.